sky/clouds/service_catalog/common.py

```diff
@@ -470,6 +471,8 @@ def list_accelerators_impl(
                                           case=case_sensitive,
                                           regex=True)]
     df['AcceleratorCount'] = df['AcceleratorCount'].astype(int)
+    if quantity_filter:
```

Comment:
Good to be consistent with other filters, eg at L469 and L466
```suggestion
    if quantity_filter is not None:
```